tests/module/utils/test_deprecation.py

```diff
@@ -7,20 +7,11 @@
 from manim.utils.deprecation import deprecated, deprecated_params
 
 
-def _get_caplog_record_msg(warn_caplog_manim):
-    logger_name, level, message = warn_caplog_manim.record_tuples[0]
+def _get_caplog_record_msg(manim_caplog):
+    logger_name, level, message = manim_caplog.record_tuples[0]
```

Comment:
Thoughts on making this return `(level, message)` so that we can make sure `level >= logging.WARNING`?